File: python/questao5.py
cigarros_por_dia = int(input("Digite a quantidade de cigarros fumados por dia: "))

anos_fumados = int(input("Digite quantos anos voce ja fumou: "))

minutos_perdidos = cigarros_por_dia * 10 * anos_fumados * 365

dias_perdidos = minutos_perdidos / 1440

print(f"Voce perdera aproximadamente {dias_perdidos:.0f} dias de vida devido ao tabagismo.")

File: c/questao5.c
#include <stdio.h>

int main() {
    int cigarrosPorDia, anosFumados;

    printf("Digite a quantidade de cigarros fumados por dia: ");
    scanf("%d", &cigarrosPorDia);

    printf("Digite quantos anos voce ja fumou: ");
    scanf("%d", &anosFumados);

    int minutosPerdidos = cigarrosPorDia * 10 * anosFumados * 365;

    int diasPerdidos = minutosPerdidos / 1440;

    printf("Voce perderá aproximadamente %d dias de vida devido ao tabagismo.\n", diasPerdidos);

    return 0;
}
